streaming/src/format/streaming.fbs

```diff
@@ -0,0 +1,37 @@
+namespace ray.streaming.fbs;
```

Comment:
Please move these files into `src/ray/streaming`.

Reply:
fixed it, and we'll replace flatbuffer with protobuffer.